package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nimiq/nano-api",
-  "version": "0.0.10",
+  "version": "0.2.0-beta.0",
```

Comment:
Any reason for this jump in version?

Reply:
I used 0.1.* for TS port versions.

Reply:
What's wrong with 0.0.11 then? As this is still the base version.

Reply:
We are changing the public API here, so this is clearly not a patch release.

Reply:
Good time to start with semver here :laughing:

Reply:
^^